core/src/franchise/franchise/franchise.service.ts

```diff
@@ -1,65 +1,80 @@
-import type {FindOneOptions} from "typeorm";
-import {Repository} from "typeorm";
+import { forwardRef, Inject, Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import type { CoreEndpoint, CoreOutput } from "@sprocketbot/common";
+import type { FindOneOptions } from "typeorm";
+import { Repository } from "typeorm";
 
-import type {FranchiseProfile} from "../../database";
-import {Franchise} from "../../database";
-import {MledbPlayerService} from "../../mledb";
+import type { FranchiseProfile } from "../../database";
+import { Franchise } from "../../database";
+import { MledbPlayerService } from "../../mledb";
+import { MemberService } from "../../organization";
 
 @Injectable()
 export class FranchiseService {
-    constructor(
-        @InjectRepository(Franchise) private readonly franchiseRepository: Repository<Franchise>,
-        @Inject(forwardRef(() => MledbPlayerService)) private readonly mledbPlayerService: MledbPlayerService,
-    ) {}
+  constructor(
+    @InjectRepository(Franchise)
+    private readonly franchiseRepository: Repository<Franchise>,
+    @Inject(forwardRef(() => MledbPlayerService))
+    private readonly mledbPlayerService: MledbPlayerService,
+    private readonly sprocketMemberService: MemberService
+  ) {}
 
-    async getFranchiseProfile(franchiseId: number): Promise<FranchiseProfile> {
-        const franchise = await this.franchiseRepository.findOneOrFail({
-            where: {id: franchiseId},
-            relations: ["profile"],
-        });
-        return franchise.profile;
-    }
-
-    async getFranchiseById(franchiseId: number): Promise<Franchise> {
-        return this.franchiseRepository.findOneOrFail({where: {id: franchiseId} });
-    }
+  async getFranchiseProfile(franchiseId: number): Promise<FranchiseProfile> {
+    const franchise = await this.franchiseRepository.findOneOrFail({
+      where: { id: franchiseId },
+      relations: ["profile"],
+    });
+    return franchise.profile;
+  }
 
-    async getFranchise(query: FindOneOptions<Franchise>): Promise<Franchise> {
-        return this.franchiseRepository.findOneOrFail(query);
-    }
+  async getFranchiseById(franchiseId: number): Promise<Franchise> {
+    return this.franchiseRepository.findOneOrFail({
+      where: { id: franchiseId },
+    });
+  }
 
-    async getPlayerFranchises(userId: number): Promise<CoreOutput<CoreEndpoint.GetPlayerFranchises>> {
-        const mlePlayer = await this.mledbPlayerService.getMlePlayerBySprocketUser(userId);
+  async getFranchise(query: FindOneOptions<Franchise>): Promise<Franchise> {
+    return this.franchiseRepository.findOneOrFail(query);
+  }
 
-        const playerId = mlePlayer.id;
+  async getPlayerFranchises(
+    memberId: number
+  ): Promise<CoreOutput<CoreEndpoint.GetPlayerFranchises>> {
+    const member = await this.sprocketMemberService.getMemberById(memberId);
+    const userId = member.userId;
```

Comment:
Not blocking but could do

```suggestion
    const { userId } = member;
```